_episodes/03-configuration.md

```diff
@@ -3,91 +3,158 @@ title: "Configuration"
+Let's change our working directory to ESMValTool, then make a copy of the file and rename it to ``config-user.yml``, then run a text editor called Nano to open it:
 
-~~~
+~~~bash
+  cd ESMValTool
   cp config-user-example.yml config-user.yml
+  nano config-user.yml
 ~~~
-{: .source}
 
 This file contains the information for:
-  * Rootpaths to the data from different projects
-  * Directory structure for input data
-  * Number of available CPUs
-  * Destination directory
-  * Auxiliary data directory
-  * Output settings
-
-## Rootpaths to input data
-ESMValTool uses several categories (in ESMValTool, this is referred to as projects) for input data based on their source, like CMIP for dataset from climate model intercomparison project, and OBS for observational dataset that adhere to (CMOR standard)[https://cmor.llnl.gov/].
-For each category, you can define either one path or several pathes as a list.
-In this lesson, you work with data from (CMIP5)[https://esgf-node.llnl.gov/projects/cmip5/].
-Add the root path of the folder where you downloaded the data during the (Setup)[https://escience-academy.github.io/lesson-esmvaltool/setup.html].
 
+* Rootpath to input data
+* Directory structure for the data from different projects
+* Number of parallel tasks
+* Destination directory
+* Auxiliary data directory
+* Output settings
+
+> ## Which text editor
+>
+> No matter what editor you use, you will need to know where it searches for and saves files. If you start it from the shell, it will (probably) use your current working directory as its default location. We use ``nano`` in examples because it is one of the least complex text editors. Press <kbd>ctrl</kbd> + <kbd>O</kbd> to save the file, and then <kbd>ctrl</kbd> + <kbd>X</kbd> to exit ``nano``.
+{: .callout}
+
+## Rootpath to input data
+
+ESMValTool uses several categories (in ESMValTool, this is referred to as projects) for input data based on their source.
+The current categories in the configuration file are mentioned below. For example, CMIP is used for a dataset from the climate model intercomparison project whereas OBS for an observational dataset. The ``rootpath`` specifies the directories where ESMValTool will look for input data. For each category, you can define either one path or several paths as a list.
+
+~~~YAML
+rootpath:
+  CMIP3: [~/cmip3_inputpath1, ~/cmip3_inputpath2]
+  CMIP5: [~/cmip5_inputpath1, ~/cmip5_inputpath2]
+  CMIP6: [~/cmip6_inputpath1, ~/cmip6_inputpath2]
+  OBS: ~/obs_inputpath
+  OBS6: ~/obs6_inputpath
+  obs4mips: ~/obs4mips_inputpath
+  ana4mips: ~/ana4mips_inputpath
+  native6:  ~/native6_inputpath
+  RAWOBS: ~/rawobs_inputpath
+  default: ~/default_inputpath
 ~~~
+
+In this lesson, we will work with data from [CMIP5](https://esgf-node.llnl.gov/projects/cmip5/).
+Add the root path of the folder where you downloaded the data during the [Setup](https://esmvalgroup.github.io/tutorial/setup.html).
```

Comment:
This assumes that people will be working locally, instead of somewhere like Jasmin, where the data will be available already. 

In the long term, a local installation is useful, but which version do we want to teach in the tutorial? Local or Jasmin? @valeriupredoi @bouweandela , thoughts?

Alternatively, do we want to write this documentation twice? Once for local and once for jasmin?

Reply:
Maybe we can make the tutorial for a local installation and add commented out versions of the correct drs settings to config-user-example.yml, so people can just uncomment the lines they need if they happen to be on a cluster that is connected to an ESGF node?

Reply:
@ledm @bouweandela I agree with Bouwe to make the tutorial for local use. I updated this part of the lesson. Now the instruction on how to get the data (or its rootpath) will come in `setup` section of the tutorial. This is the most common approach for several lessons that I have seen in software carpentry.  The `setup` section contains everything related to data and additional material. Therefore, the episode content won't be changed.

Reply:
Created an issue to do this: https://github.com/ESMValGroup/ESMValTool/issues/1622